rpm/SPECS/git-lfs.spec

```diff
@@ -6,7 +6,7 @@ Summary:        Git extension for versioning large files
 Group:          Applications/Archiving
 License:        MIT
 URL:            https://git-lfs.github.com/
-Source0:        https://github.com/github/git-lfs/archive/%{name}-%{version}.tar.gz
+Source0:        https://github.com/github/git-lfs/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
but rpm needs last path component to be filename {name}-{version}.tar.gz, therefore the url hack. and it works fine.

you can inject any garbage after /archive/tag in url, github will ignore the junk and provide data under the complete url.